src/gui/about.qrc

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE RCC><RCC version="1.0">
-<qresource>
-  <file>gpl.html</file>
-  <file>thanks.html</file>
-  <file>translators.html</file>
-</qresource>
+    <qresource>
```

Comment:
Did you make these indentation changes manually?

Reply:
yes, what about it?

Reply:
I'm just worried that QtCreator (or Qt Designer) will change it back...

Reply:
I just tried QtCreator, it respects the indention, so no problem about it.